tests/unit/test_ktp_api.py

```diff
@@ -30,17 +29,17 @@
 ]
 
 tracy_jobs_list = [
-    {"id": 1, "args": "arguments", "function": "TRAIN_SPECIFIC_MODEL"},
+    {"id": 1, "args": "arguments", "function": "TRAIN_MODEL"},
     {
         "id": 2,
         "args": "arguments2",
-        "function": "OPTIMIZE_HYPERPARAMETERS_FOR_SPECIFIC_PID",
+        "function": "OPTIMIZE_HYPERPARAMETERS",
```

Comment:
Is this an additional bugfix, or is it related to this PR.
In any case; good idea to solve it.

Reply:
yeah, this is an additional bugfix that slipped though a recent PR I think.